src/main/java/uk/ac/cam/cl/dtg/isaac/dto/AssignmentDTO.java

```diff
@@ -17,14 +17,15 @@
 
 import java.util.Date;
 
+import uk.ac.cam.cl.dtg.isaac.api.services.EmailService;
 import uk.ac.cam.cl.dtg.segue.dto.users.UserSummaryDTO;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 /**
  * This class is the Data Transfer Object used to store Assignments in the isaac CMS.
  */
-public class AssignmentDTO {
+public class AssignmentDTO implements EmailService.AssignmentLike {
```

Comment:
Might it be better as an `AbstractAssignmentDTO` that the classic assignments and quiz assignments both implement in the content, to avoid coupling content DTO classes to something outside the DTO package?
I can see that that approach won't work for `HasTitleOrId` though. Perhaps it also already happens and it doesn't matter.

(I don't know if @sacummins will see this in time, or have any particular opinions?)

Reply:
FWIW, in the third PR I moved `EmailService.AssignmentLike` to `IAssignmentLike` in the DTO package. I don't know if an abstract base would be better. There may be a more Java-y way of doing this but I thought the interface was alright.

Reply:
Thanks @mlt47 I think my next branch moves it so hopefully that all makes sense.